DOCS.md

```diff
@@ -188,6 +188,10 @@ vars
 : a map of variables to pass to the Terraform `plan` and `apply` commands.
 Each value is passed as a `-var <key>=<value>` option.
 
+var_files
+: a list of variable files to pass to the Terraform `plan` and `apply` commands.
```

Comment:
The logic for the plugin will always use the plan options and conditionally use the apply options. This way you can add options relevant to both to just the plan stage.

So by adding it to plan it becomes available in both plan and apply.